solutions/swb-reference/integration-tests/config/testEnv.yaml

```diff
@@ -15,10 +15,6 @@ terminatedEnvId: 'env-b44262fb-ed34-47ae-a4a4-d2d9d9c7a5d5' # id of an environme
 rootUserNameParamStorePath: '/swb/testEnv/rootUser/email'
 rootPasswordParamStorePath: '/swb/testEnv/rootUser/password'
 
-## For running project integration tests
-costCenterId: 'cc-90c8466a-d7d9-4da8-a128-07eb270487f0'
```

Comment:
Could you also see if there are any other places where we use costCenterId? I recently added it here: https://github.com/aws-solutions/solution-spark-on-aws/blob/develop/solutions/swb-reference/integration-tests/tests/multiStep/userProject.test.ts#L18

Reply:
oh no this comment didn't show up until after I hit merge :-|